build/ci/save_driver_pods_logs.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash -x
+
+run_action_on_pod (){
+    pod_name=$1
+    action=$2
+    extra_args=$3
+    get_all_pods_by_type="kubectl get pod -l csi | grep $pod_name | awk '{print$1}'"
+    kubectl $action $($get_all_pods_by_type) $extra_args -c ibm-block-csi-$pod_name > /tmp/driver_$($get_all_pods_by_type)_$action.txt
+}
+
+declare -a pod_types=(
+    "node"
+    "controller"
+    "operator"
+)
+
+for pod_type in "${pod_types[@]}"
```

Comment:
is it [name](https://github.com/IBM/ibm-block-csi-operator/blob/bba693e2dc12819d16880f499f9b92a36ea41ff3/build/ci/save_driver_pods_logs.sh#L4) or type?

Reply:
name

Reply:
then why `type`?

Reply:
oops *type
I meant to type

Reply:
then why [pod_name](https://github.com/IBM/ibm-block-csi-operator/blob/bba693e2dc12819d16880f499f9b92a36ea41ff3/build/ci/save_driver_pods_logs.sh#L4)?